app/models/saved_claim/dependency_claim.rb

```diff
@@ -134,6 +134,29 @@ def to_pdf(form_id: FORM)
     PdfFill::Filler.fill_form(self, nil, { created_at: })
   end
 
+  def send_failure_email(encrypted_user_struct = nil)
```

Comment:
Can you please add a docblock here and some comments that reflect that this isn't the ideal way we wanted to go about it.  
Basically, make it clear to anyone following on behind that if we had more time, we would have done this using the common lib:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/va_notify/notification_email.rb

Reply:
done!